frontend/model/notifications/templates.js

```diff
@@ -9,39 +9,30 @@ import { L, LTags } from '@common/common.js'
 import { humanDate } from '@model/contracts/shared/time.js'
 import {
   STATUS_PASSED, STATUS_FAILED, STATUS_CANCELLED, STATUS_EXPIRED,
-  PROPOSAL_INVITE_MEMBER, PROPOSAL_REMOVE_MEMBER,
+  PROPOSAL_INVITE_MEMBER, PROPOSAL_REMOVE_MEMBER, CHATROOM_MEMBER_MENTION_SPECIAL_CHAR,
   PROPOSAL_GROUP_SETTING_CHANGE, PROPOSAL_PROPOSAL_SETTING_CHANGE, PROPOSAL_GENERIC
 } from '@model/contracts/shared/constants.js'
 import { getProposalDetails } from '@model/contracts/shared/functions.js'
 
-const contractName = (contractID) => sbp('state/vuex/state').contracts[contractID]?.type ?? contractID
-const userDisplayNameFromID = (userID) => sbp('state/vuex/getters').userDisplayNameFromID(userID)
-// Note: this escaping is not intended as a protection against XSS.
-// It is only done to enable correct rendering of special characters in usernames.
-// To guard against XSS when rendering usernames, use the `v-safe-html` directive.
-const escapeForHtml = (text) => text.replace(/[<>&]/g, (c) => ('&#' + c.codePointAt(0) + ';'))
-const strong = (text) => `<strong>${escapeForHtml(text)}</strong>`
-
 export default ({
   CHELONIA_ERROR (data: { activity: string, error: Error, message: GIMessage }) {
     const { activity, error, message } = data
     const contractID = message.contractID()
     const opType = message.opType()
     const value = message.decryptedValue()
     let action
-    let meta
     if ([GIMessage.OP_ACTION_ENCRYPTED, GIMessage.OP_ACTION_UNENCRYPTED].includes(opType) && value) {
       action = value.action
-      meta = value.meta
     }
+
     return {
       body: L("{errName} during {activity} for '{action}' from {b_}{who}{_b} to '{contract}': '{errMsg}'", {
         ...LTags('b'),
         errName: error.name,
         activity,
         action: action ?? opType,
-        who: meta?.username ?? message.signingKeyId(),
-        contract: contractName(contractID),
+        who: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${message.signingKeyId()}`,
```

Comment:
Please note: the `message.signingKeyId()` is **NOT** a `contractID`. So you cannot use it to get the username and therefore `CHATROOM_MEMBER_MENTION_SPECIAL_CHAR` is also inappropriate here.

In fact it seems there's currently no simple way to figure out this information from a `GIMessage`, and not all `GIMessage`s contain this information.

So I've created #2125 to address this and assigned it to @corrideat, but for now let's just remove this `who` variable altogether.